gammapy/estimators/points/tests/test_core.py

```diff
@@ -255,6 +255,33 @@ def test_plot_likelihood_error(self, flux_points_likelihood):
             flux_points_likelihood.plot_ts_profiles(ax=ax)
 
 
+@requires_data()
+def test_plot_format_yaxis():
+    path = "$GAMMAPY_DATA/tests/spectrum/flux_points/flux_points.fits"
+    fp = FluxPoints.read(path)
+
+    with mpl_plot_check():
+        ax = fp.plot(sed_type="e2dnde")
+        assert (
+            ax.yaxis.get_label().get_text()
+            == "e2dnde [$\\mathrm{erg\\,s^{-1}\\,cm^{-2}}$]"
+        )
+
+    with mpl_plot_check():
+        ax = fp.plot(sed_type="dnde", energy_power=2)
```

Comment:
Could you check what happens with a weird power like 2.7?

Reply:
Working well !
![fp_epower2 7](https://github.com/user-attachments/assets/9e56aa00-d6db-4141-8f36-69c2c7a2a74b)